commit fb591ff90a04f2e777f4897b606c90f6d68aafec
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Wed Nov 7 17:43:10 2018 +0000

    overlays: uart0 - return GPIOs 14 and 15 to inputs
    
    In the event that alternate pins are used (only useful on Compute
    Modules), return the standard pins to inputs to avoid double-mapping
    them.
    
    See: https://www.raspberrypi.org/forums/viewtopic.php?p=1388713#p1316977
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/boot/dts/overlays/uart0-overlay.dts b/arch/arm/boot/dts/overlays/uart0-overlay.dts
index 20b2a609c511..7df217e6f16d 100755
--- a/arch/arm/boot/dts/overlays/uart0-overlay.dts
+++ b/arch/arm/boot/dts/overlays/uart0-overlay.dts
@@ -17,16 +17,17 @@
 		target = <&gpio>;
 		__overlay__ {
 			uart0_pins: uart0_pins {
-				brcm,pins = <14 15>;
-				brcm,function = <4>; /* alt0 */
-				brcm,pull = <0 2>;
+				brcm,pins = <14 15 14 15>;
+				brcm,function = <0 0 4 4>; /* alt0 */
+				brcm,pull = <0 0 0 2>;
 			};
 		};
 	};
 
 	__overrides__ {
-		txd0_pin = <&uart0_pins>,"brcm,pins:0";
-		rxd0_pin = <&uart0_pins>,"brcm,pins:4";
-		pin_func = <&uart0_pins>,"brcm,function:0";
+		txd0_pin = <&uart0_pins>,"brcm,pins:8";
+		rxd0_pin = <&uart0_pins>,"brcm,pins:12";
+		pin_func = <&uart0_pins>,"brcm,function:8",
+			   <&uart0_pins>,"brcm,function:12";
 	};
 };
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1653_fb591ff90a04f2e777f4897b606c90f6d68aafec.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1653_fb591ff90a04f2e777f4897b606c90f6d68aafec.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
